src/assets/styles/sass/styles.scss

```diff
@@ -0,0 +1,87 @@
+@mixin project-bg {
+    background-size: contain;
+    height: 90%;
+    width: 100%;
+    display: grid;
+    grid-template: 5fr 0.5fr / 1fr;
+    place-items: center;
+}
+
+.projects-listing {
+display: grid;
+grid-template: repeat(2, 1fr) / repeat(3, 1fr);
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/descholar-ceo/portfolio/pullRequest?prid=5731031)